problems/5.loops.js

```diff
@@ -1,33 +1,55 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
-
-}
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+//function evenNumbers() {}
+const evenNumbers = (num) => {
+  let evenArray = [];
+  for (i = 0; i <= num; i += 2) {
+    if (i % 2 === 0) {
+      evenArray.push(i);
+    }
+  }
+  return evenArray;
+};
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
-* @param {number} num - The number to find multiples of
-* @returns {number[]} - An array of all multiples of {num} up to 1000
-*/
-function multiplesToOneThousand() {
-
-}
+ * @param {number} num - The number to find multiples of
+ * @returns {number[]} - An array of all multiples of {num} up to 1000
+ */
+//function multiplesToOneThousand() {}
+const multiplesToOneThousand = (num) => {
+  let sum = 0;
+  let multiplesArr = [0];
+  loop = Math.floor(999 / num);
```

Comment:
Always declare varaibles otherwise they're global

Reply:
accident with that one.